.github/workflows/build-images.yaml

```diff
@@ -109,8 +109,9 @@ jobs:
       - name: Set Operator version
         id: operator-version
         run: |
-          if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
-          echo "VERSION=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
+          tag=${{GITHUB_REF_NAME}}
```

Comment:
`GITHUB_REF_NAME` often contains slashes, such as in the case of branch names like `docs/foo`.

Reply:
yup, but it would be the same for the if condition, no?

Reply:
I mean when we set the version to it, we need to replace all slashes with another char, as the slash symbol is not allowed in the manifests, I recon.

Reply:
so on 114 we need to add the string replace? 
`/\//-` ?

Reply:
Or add it on 112 and remove the leading "v" in the regex in 113.

Reply:
I'd say it's not needed, since `VERSION` assignment happens within 2 branches of the conditional. First case matching the REGEX, SemVer with leading "v" and beta tags (as in `v0.9.0-cr1`) and the second when doesn't match it assigns the ref sha.

Reply:
Understood, @didierofrivia. Thanks for clarifying.